src/lib.rs

```diff
@@ -143,22 +143,20 @@ pub fn test(_attr: TokenStream, item: TokenStream) -> TokenStream {
 ///
 /// ```ignore
```

Comment:
test was ignored

Reply:
Ah dang. Yep definitely missed that.

Reply:
Oh, doc tests run with `cargo build`, so it seems that we can't actually test `#[test]` and `#[bench]` via doc tests...